src/Discord.Net.Rest/Net/Queue/RequestQueueBucket.cs

```diff
@@ -250,6 +250,10 @@ private void UpdateRateLimit(int id, RestRequest request, RateLimitInfo info, bo
                 else if (info.Reset.HasValue)
                 {
                     resetTick = info.Reset.Value.AddSeconds(info.Lag?.TotalSeconds ?? 1.0);
+
+                    if (request.Options.IsReactionBucket)
+                        resetTick = new DateTimeOffset(resetTick.Value.Ticks / 4, TimeSpan.Zero);
```

Comment:
Isn't this dividing the actual unix timestamp by 4? It looks like you actually mean to divide the `diff` value below by 4 instead